lib/srv/db/cloud/watchers/rds.go

```diff
@@ -142,6 +142,10 @@ func (f *rdsFetcher) getAuroraDatabases(ctx context.Context) (types.Databases, e
 	}
 	databases := make(types.Databases, 0, len(clusters))
 	for _, cluster := range clusters {
+		if !services.IsRDSClusterSupported(cluster, f.log) {
+			continue
+		}
```

Comment:
I'd return an error here and log before the continue rather than require a logger to be passed as an argument. 

I see that @r0mant had suggested we remove the error here and log/continue instead, but the code has since changed from `services.NewDatabaseFromRDSCluster(cluster) ` to use this very simple and clean `services.IsRDSClusterSupported`. Logging is the responsibility of `rdsFetcher`, which would remove the side effect from `IsRDSClusterSupported`.

Sorry to go back and forth on this part.

Reply:
Yeah, I was thinking the same, it's no big deal either way but logging here would be a little cleaner and you wouldn't need to pass log parameter to the method.

```suggestion
		if !services.IsRDSClusterSupported(cluster) {
		        f.log.Debugf("Aurora cluster %q (engine mode %v, engine version %v) doesn't support IAM authentication. Skipping.",
		            aws.StringValue(cluster.DBClusterIdentifier),
		            aws.StringValue(cluster.EngineMode),
			    aws.StringValue(cluster.EngineVersion))
			continue
		}
```